_data/partners.yml

```diff
@@ -200,4 +200,31 @@ partners:
   - title: "Buncombe County Health & Human Services"
     image: "partner-buncombe-county-health-human-services.png"
     link: "https://www.buncombecounty.org/Governing/Depts/HHS/Default.aspx"
-    include-page: ["home-partners", "strategic-partners"]
\ No newline at end of file
+    include-page: ["home-partners", "strategic-partners"]
+  - title: "WNCAP"
+    image: "partner-wncap.png"
+    link: "https://wncap.org/"
+    include-page: ["home-partners", "strategic-partners"]
+  - title: "12 Baskets"
+    image: "partner-12-baskets.png"
+    link: "https://www.ashevillepovertyinitiative.org/"
+    include-page: ["home-partners", "strategic-partners"]
+  - title: "Julian F. Keith Alcohol and Drug Abuse Treatment Center"
+    image:
```

Comment:
We definitely need an image asset here, I think.

Reply:
Oh, I see - you altered the section template for when we lack an image asset?

Reply:
Correct -- manifest said there was no image, and to just link the text.